invenio_jsonschemas/__init__.py

```diff
@@ -22,7 +22,17 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""Invenio module for building and serving JSONSchemas."""
+"""Invenio module for building and serving JSONSchemas.
```

Comment:
2017? :)

Reply:
Good point, I moved the text but forgot about the license :P